framework-plugins/lisk-framework-report-misbehavior-plugin/src/report_misbehavior_plugin.ts

```diff
@@ -43,7 +48,7 @@ import * as controllers from './controllers';
 
 // eslint-disable-next-line
 const packageJSON = require('../package.json');
-const logger = debug('plugin:resport-misbehavior');
+const logger = debug('plugin:report-misbehavior');
```

Comment:
Logger makes it confusing with the file logger. Once see it in the code may look for the relevant logs in the console or log file. Better to rename it to `debug` which is convention we used other places as well.